src/cli/README_UDP.md

```diff
@@ -96,14 +96,22 @@ Done
 
 Specifies the peer with which the socket is to be associated.
 
-- ip: the peer's IPv6 address.
+- ip: the peer's IP address.
 - port: the peer's UDP port.
 
 ```bash
 > udp connect fdde:ad00:beef:0:bb1:ebd6:ad10:f33 1234
 Done
 ```
 
+> Note: You may use an IPv4 address, which will be mapped to IPv6 address using the preferred NAT64 prefix from the network data.
```

Comment:
Suggestion: mention the feature flags required to support IPv4?

Reply:
We don't need feature flags on the client side. It can be used as long as we have a valid NAT64 prefix in the network data.

Reply:
got it.